ci: add workflow for the Bounty Program commands

aeworxet opened the pull request:
This PR adds GitHub Actions workflow to allow [Bounty Team members](https://github.com/orgs/asyncapi/teams/bounty_team) to issue commands to the organization's global AsyncAPI bot related to the Bounty Program, while at the same time preventing unauthorized users from misusing them.

This particular workflow adds a command that allows adding the label `bounty` to GitHub issues, thereby making them Bounty Issues.
Command is invoked on creation of comment `/bounty`.

derberg commented on .github/workflows/bounty-program-commands.yaml:
```diff
@@ -0,0 +1,89 @@
+        with:
+          github-token: ${{ secrets.GH_TOKEN }}
+          script: |
+            const commentText = `❌ @${{github.actor}} is not authorized to use a Bounty Program's command.`;
```
maybe we should mention you and Thulie in comment
- so it is clear who is authorized
- so you are automatically pinged and see that people want to use it

derberg commented on .github/workflows/bounty-program-commands.yaml:
```diff
@@ -0,0 +1,89 @@
+
+        with:
+          github-token: ${{ secrets.GH_TOKEN }}
+          script: |
```
shouldn't we get a confirmation comment from bot here as well, that bounty addition was successful?
or is it enough that basically label will be added?

aeworxet replied:
It will be observed in real-time whether the label is added or not; nothing more is needed or expected from the bot.

derberg reviewed (commented):
lgtm, please add it to https://github.com/asyncapi/.github/blob/master/.github/workflows/global-replicator.yml#L130 as this is a workflow that must end up in every repository.

and yeah, please when you add a change, do not do force push, it is not helping in review 😅

aeworxet commented:
> please add it to https://github.com/asyncapi/.github/blob/master/.github/workflows/global-replicator.yml#L130

- renamed `.yaml` to `.yml`
- added

derberg commented:
/rtm
